src/Windows/R/Components/Impl/InteractiveWorkflow/Implementation/RInteractiveWorkflow.cs

```diff
@@ -72,16 +68,16 @@ public sealed class RInteractiveWorkflow : IRInteractiveWorkflowVisual {
             History = historyProvider.CreateRHistory(this);
             Packages = packagesProvider.CreateRPackageManager(_settings, this);
             Plots = plotsProvider.CreatePlotManager(_settings, this, coreShell.FileSystem());
-            _operations = new RInteractiveWorkflowOperations(this, _debuggerModeTracker, Shell);
+            _operations = new RInteractiveWorkflowOperations(this, debuggerModeTracker, Shell);
 
-            _activeTextViewTracker.LastActiveTextViewChanged += LastActiveTextViewChanged;
+            activeTextViewTracker1.LastActiveTextViewChanged += LastActiveTextViewChanged;
```

Comment:
Why do you need another variable instead of `activeTextViewTracker`?

Reply:
Resharper change lol